api/proto/teleport/identitycenter/v1/identitycenter.proto

```diff
@@ -34,6 +34,10 @@ message PermissionSetInfo {
   // Role is an optional ARN indicating role provisioned to this account. May be
   // empty if the permission set is not provisioned or is not relevant in the
   string role = 3;
+
+  // AssignmentName is the name of an account assignment representing this
+  // permission set being assigned to the enclosing account.
+  string assignment_name = 4;
```

Comment:
What is this "assignment name", is this the name of some Teleport internal object or something else? Also, a permission set can be assigned to multiple accounts - is this going to be an issue, or is this PermissionSetInfo is supposed to be used within an account object it's assigned to?

Reply:
Added explanatory comments